### PR TITLE
Update reason for duplicates checking

### LINKED ISSUE #158: Update UG notes for Duplicate restrictions

### PR DESCRIPTION
**Pull request by DennieDan**
Closes #158 #171 #198 
This PR
- Updates reason for duplicates checking
- Change the box of **Features** from `seamless` to normal. Can view the following pic and let me know if you prefer this or the old one
image
- Add `exit` command to Command Summary

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -70,7 +72,7 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
 
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+Shows a message explaining how to access the help page.
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:75]: Good catch👍. Thanks for changing this.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -49,18 +49,20 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
...
 
-<box type="info" seamless>**Notes about the command format:**<br>
+<box type="info"><md>**Notes about the command format:**</md><br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+  e.g `n/NAME [owe/OWED_AMOUNT]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:58]: Thanks for changing this too

**Review by Soumyadip-cmd — APPROVED** [review]
LGTM. Thanks for spotting the mistakes and changing them. You could perhaps put the bug label on this PR.

**Comment by DennieDan** [comment]
> LGTM. Thanks for spotting the mistakes and changing them. You could perhaps put the bug label on this PR.

Sure, I will

File: docs/UserGuide.md (hunk 2 above)
Comment (Feng1231) [inline docs/UserGuide.md:58]: hmm, t/TAG is removed be the examples not updated yet! 😢 
> `n/John Doe t/friend` or as `n/John Doe`.

These are still using PREFIX_TAG, right?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -49,18 +49,20 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
...
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
-* No two students can have both same **NAME** and **PHONE**.
+* No two students can have both same **NAME** and **PHONE**, but different students may share a **PHONE** number.
+
+> Reason: Siblings can use a parent's phone number as their **PHONE**. 
 
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
```
Comment (Feng1231) [inline docs/UserGuide.md:67]: This part should include all the new commands too.
income, remind, and others if I miss out any.
Perhaps we should create another issue for it

File: docs/UserGuide.md [hunk 4]
```diff
@@ -49,18 +49,20 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
...
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+  e.g `n/NAME [owe/OWED_AMOUNT]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
-* No two students can have both same **NAME** and **PHONE**.
+* No two students can have both same **NAME** and **PHONE**, but different students may share a **PHONE** number.
```
Comment (Feng1231) [inline docs/UserGuide.md:63]: the parameters provided is PHONE_NUMBER in this case, should we just standardize to just refer to **PHONE_NUMBER**

**Review by Feng1231 — CHANGES_REQUESTED** [review]
Nice, but need to update the example too

**Review by Feng1231 — APPROVED** [review]
LGTM

**Pull request merged**